test/hotspot/jtreg/runtime/logging/ClassLoadUnloadTest.java

```diff
@@ -42,6 +42,7 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Set;
```

Comment:
This seems to be unused. Or perhaps should check the return `Set<String>` from the call to `ClassUnloadCommon.triggerUnloading` below as is being done in other tests?

Reply:
The test didn't check that the className was unloaded with WhiteBox only with the logging message, so we don't need the return Set type.  I removed this import.